### ISSUE TITLE
[Feature Request] - adding mailbox by department owner

### ISSUE THREAD

**Issue by ghost** [osTicket/osTicket#5179]
Got the following privacy issue:
I have 6 different departments all with its own mailbox. I do not wat all department-members or department-"leaders" to have admin rights. So an admin has to add the mailbox to the ticketsystem.

Here's my problem : in order to add a departments mailbox , i have to now the credentials .... wich i may defenatly may not.  I also do not want to add admin rights for a brief moment to this department or department-member.

There is a way and i hope this can be implemented in order to meet the GPDR(AVG) standards:
Why not give a team-member specific rights (lets call it super user/agent) in order to add their own mailbox? This way i do not know the credentials (wich i may not know for sure).

**Comment by JediKev** [comment]
@bernardflach 

Unfortunately, I don't think this will be a thing. The way you setup emails in osTicket is you have configure the Fetching/Sending credentials in Admin Panel > Emails > Emails. Once configured you can then set a specific Department for tickets that come in via the Address in the mail configurations. In order to set the newly configured email as the Outgoing Email for the Department you will need to go to the Department and configure the Outgoing Email option to the new email.

>in order to meet the GPDR(AVG) standards

As I've heard it here before, my understanding is that the GPDR responsibility falls on the installer, not the software itself.

Cheers.

**Comment by ghost** [comment]
> As I've heard it here before, my understanding is that the GPDR responsibility falls on the installer, not the software itself.

Partially ... also software has to support GPDR in order to protect the agents and their info (in this case mailbox credentials). There is allways a way around this (using own mailboxes) but some departments want their own mailbox and identity so now i have to add the mailboxes using their credentials. This is not wanted .. do you see the "problem" ?

**Comment by JediKev** [comment]
@bernardflach 

Not really, you can add the mailbox yourself or give the Agent temporary access to add their mailbox.

Cheers.

**Comment by ghost** [comment]
the mailbox is owned by the other department .... and i don't ask for credentials of their mailbox ... it is the same as lending your atm card and giving the pin code. 
You don't

Now indeed i have to give soneone ALL access to admin in order to just ad their mailbox ....it is a choice .. but not myprimary one. this way an agent has temporarly access to data he/she is not allowed to...

**Comment by ghost** [comment]
there is a simple way ... make an extra choice on editiing users ... besides making admin , create a make super user/agent witch allows him/her to just only the adding mailbox function

**Comment by ghost** [comment]
perhaps there is a way to discuss this not using git ... its a complicated matter , with some simple solutions ;)

**Comment by JediKev** [comment]
@bernardflach 

We don't have the possibility of restricting Agents to just one page/setting in the admin panel...either you are an Admin or you aren't. So I don't think this will be a viable option imo.

Cheers.

**Comment by ghost** [comment]
@JediKev : correct , there is not at this moment ... as you have lots of EU users , you **will** get this questions more .... in EU you will have to explain why you make some disicions...(transparency)  and its not allowed to as for credentials ... as i mentioned before .. you don't.

Please take it in consideration (perhaps together with @protich) as this (privacy-thing) is a **big** issue in the EU.  Thanks and cheers  ;)

**Comment by protich** [comment]
@JediKev - it's a feature request! We should consider granular permissions when we deal with multi-brand feature - which i think makes sense for this use case.

**Comment by ghost** [comment]
@JediKev and @protich ...thanks for the attention and reply's ... i will follow this ;)
Have a nice weekend (y)
